arbitrum-docs/bold/bold-gentle-introduction.md

```diff
@@ -0,0 +1,139 @@
+- BOLD is currently considered to be in `alpha` release and is deployed on a public testnet. [Follow this guide](https://github.com/OffchainLabs/bold-validator-starter-kit) to deploy a BOLD validator to test & explore, first hand, how BOLD works to secure Arbitrum chains. To learn more about BOLD, please check out the [BOLD whitepaper](https://github.com/OffchainLabs/bold/blob/main/docs/research-specs/BOLDChallengeProtocol.pdf) and [BOLD's code and specifications on Github](https://github.com/OffchainLabs/bold).
+
+## What _exactly_ is BOLD?
+BOLD, an acronym for Bounded Liquidity Delay Protocol, is a set of rules used by validators to open and resolve disputes about Arbitrum’s state to ensure only valid states get confirmed to an Arbitrum chain’s data availability layer, such as Ethereum, for example. BOLD is shipped as part of the Nitro tech stack - the software that powers every Arbitrum validator node. 
+
+In short, BOLD secures the process of withdrawing assets from Arbitrum back to Ethereum.
+
+A BOLD-enabled validator’s responsibilities are to:
+
+- Post claims about an Arbitrum chain’s state to Ethereum,
+- Challenge invalid claims made by other validators, and
+- Confirm valid claims - either by timing other validators out or by winning a challenge
+
+The goal of BOLD is ensure the maximum challenge period is 6.4 days, effectively removing the risk of delay attacks and making L2 to L1 withdrawals more secure. BOLD accomplishes this by introducing a new dispute system that lets a single entity secure Arbitrum against malicious parties - effectively allowing anyone to validate Arbitrum’s state without needing permission to do so.
+
+
+## Why does Arbitrum need a new dispute protocol?
+
+While Arbitrum chains today have working fraud proofs to secure withdrawals, BOLD introduces a few subtle but innovative changes that let *anyone* challenge and win disputes - all within a fixed time period. In other words, Arbitrum chains will continue to be secured with an interactive proving game between validators, but with the added benefit of this game being completely permissionless and time-bounded to 1 challenge period (currently set at 6.4 days).
+
+Under the hood, the reason why BOLD can offer time-bound, permissionless validation is because a correct Arbitrum state assertion is **not tied to a single staker or entity**. Instead, because L2 states are completely deterministic and can eventually be proven on Ethereum, any party can stake on this correct, deterministic state and, through interactive fraud proofs, can prove that their claim is correct. This means that a single honest party staking on the correct state assertion will always win, guaranteed. 
```

Comment:
"Instead, because L2 states are completely deterministic and can eventually be proven on Ethereum, any party can stake on this correct, deterministic state and, through interactive fraud proofs, can prove that their claim is correct. This means that a single honest party staking on the correct state assertion will always win, guaranteed."

This is also the case with the current challenge protocol, not a feature from Bold.